index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alusine's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <a href="index.html"><h1>My Logo</h1></a>
+      <nav>
+        <svg
+          class="menu w-6 h-6"
+          fill="none"
+          stroke="currentColor"
+          viewBox="0 0 24 24"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            d="M4 6h16M4 12h16M4 18h16"
+          ></path>
+        </svg>
+      </nav>
+    </header>
+
+    <section class="container">
+      <div class="content">
+        <div class="header2">
+          <h1>I'm Alusine, glad to meet you!</h1>
```

Comment:
Let's make this logo a link, so when a visitor clicks on it, he/she is redirected back to the home page.

Reply:
@Alusp , i am so sorry, i marked the wrong part, and you have done the project correctly. I will raise the request with microverse team about this.